integration-cli/docker_cli_run_test.go

```diff
@@ -4058,3 +4058,64 @@ func (s *DockerSuite) TestRunNamedVolumeCopyImageData(c *check.C) {
 	out, _ := dockerCmd(c, "run", "-v", "foo:/foo", "busybox", "cat", "/foo/hello")
 	c.Assert(strings.TrimSpace(out), check.Equals, "hello")
 }
+
+func (s *DockerSuite) TestRunWithPull(c *check.C) {
+	testRequires(c, Network)
```

Comment:
@vdemeester I switched to testing with the 'busybox' image, which I assume can be used for testing ARM?  I'm not sure how to trigger am arm build to test this, however.